microsoft/testsuites/core/timesync.py

```diff
@@ -115,7 +115,9 @@ def verify_timesync_ptp(self, node: Node) -> None:
         #  instead of /dev/ptp0 or /dev/ptp1.
         for chrony_config in self.chrony_path:
             if node.shell.exists(PurePosixPath(chrony_config)):
```

Comment:
@adityagesh thanks for the fix, please have one more fix for the loop, it should not assert for each file.

Reply:
Updated, can you please check if this logic is fine?